.github/workflows/check_release_version.yml

```diff
@@ -5,35 +5,23 @@ on:
   pull_request:
     branches:
       - master
-      - develop
```

Comment:
Just to check. develop is removed from here because we are now automating releases for develop?

Reply:
Yes we don't want to check whether the `.release-version` file has been incremented by hand as we'll be generating unique version in develop anyway.  But when merging `develop` to `master`, the person should also have made a commit in `develop` changing the release version.